### DISCUSSION TITLE
A proof of two pages

### DISCUSSION THREAD

**Discussion by xtexx** [community/community#140461] [GitHub Education]
### Hi everyone,

Hi.
My school issued certifications for all students. However, it puts:
- school name on the first page
- student information on the second page
- stamps on the fourth page
What should I do?

**Comment by ferid333** [comment]
You can scan or take clear photos of all the necessary pages (the ones with the school name, your information, and the stamps) and combine them into a single PDF file. There are many free tools online that allow you to merge multiple PDFs or images into one.
